app/controllers/home_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomeController < ApplicationController
+
+  def home_page
```

Comment:
this can be simply called index or show now, since this is home controller

Reply:
Yes sir fixed.